.github/scripts/get_tutorials_stats.py

```diff
@@ -0,0 +1,239 @@
+    return rc
+
+def table_exists(table_name: str) -> bool:
+    """
+    Determines whether a table exists. As a side effect, stores the table in
+    a member variable.
+    """
+    try:
+        table = dynamodb.Table(table_name)
+        table.load()
+        exists = True
+        print("Table exists")
+    except ClientError as err:
+        if err.response['Error']['Code'] == 'ResourceNotFoundException':
+            exists = False
+        else:
+            print("Unknown error")
+            pprint(err.response)
+    return exists
+
+def delete_table(table_name: str) -> None:
+    table = dynamodb.Table(table_name)
+    table.delete()
+
+    print(f"Deleting {table.name}...")
+    table.wait_until_not_exists()
+
+def create_table(table_name: str) -> None:
+    """
+    Creates a DynamoDB table.
+
+    :param dyn_resource: Either a Boto3 or DAX resource.
+    :return: The newly created table.
+    """
+
+    table_name = table_name
+    params = {
+        'TableName': table_name,
+        'KeySchema': [
+            {'AttributeName': 'commit_id', 'KeyType': 'HASH'},
+            {'AttributeName': 'date', 'KeyType': 'RANGE'}
+        ],
+        'AttributeDefinitions': [
+            {'AttributeName': 'commit_id', 'AttributeType': 'S'},
+            {'AttributeName': 'date', 'AttributeType': 'S'}
+        ],
+        'ProvisionedThroughput': {
+            'ReadCapacityUnits': 10,
+            'WriteCapacityUnits': 10
+        }
+    }
+    table = dynamodb.create_table(**params)
+    print(f"Creating {table_name}...")
+    table.wait_until_exists()
+
+def create_table2(table_name: str) -> None:
+    """
+    Creates a DynamoDB table.
+
+    :param dyn_resource: Either a Boto3 or DAX resource.
+    :return: The newly created table.
+    """
+
+    table_name = table_name
+    params = {
+        'TableName': table_name,
+        'KeySchema': [
+            {'AttributeName': 'commit_id', 'KeyType': 'HASH'},
+            {'AttributeName': 'filename', 'KeyType': 'RANGE'}
+        ],
+        'AttributeDefinitions': [
+            {'AttributeName': 'commit_id', 'AttributeType': 'S'},
+            {'AttributeName': 'filename', 'AttributeType': 'S'}
+        ],
+        'ProvisionedThroughput': {
+            'ReadCapacityUnits': 10,
+            'WriteCapacityUnits': 10
+        }
+    }
+    table = dynamodb.create_table(**params)
+    print(f"Creating {table_name}...")
+    table.wait_until_exists()
+
+def convert_to_dict(entry: Tuple[str, List[Tuple[str, int, int]]]) -> List[Dict[str, Union[str, int]]]:
+    return [
+        {'commit_id': entry[0], 'filename': i[0], 'lines_added': i[1], 'lines_deleted': i[2]}
+        for i in entry[1]
+    ]
+
+def main() -> None:
+    tutorials_dir = os.path.expanduser("./tutorials")
+    get_history_log = get_history(tutorials_dir)
+    commits_to_files = get_file_names(tutorials_dir)
+    table_name_history = 'torchci-tutorial-metadata'
+    table_name_filenames = "torchci-tutorial-filenames"
```

Comment:
Typically, global constants are defined at the start of the file and in all caps and in polish notiation, i.e. `HISTORY_TABLE_NAME='torchci-tutorials-metadata'`